test/integration/remote-config.spec.ts

```diff
@@ -0,0 +1,259 @@
+/*!
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as admin from '../../lib/index';
+import * as chai from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+import { deepCopy } from '../../src/utils/deep-copy';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+const expect = chai.expect;
+
+const VALID_PARAMETERS: { [key: string]: admin.remoteConfig.RemoteConfigParameter } = {
```

Comment:
Can you also include some randomly generated value somewhere in these inputs? Otherwise a new test session, might pick up the changes made in a previous test run.

Reply:
Good catch. Thanks!